### PR TITLE
fixes bug 1214436 - Ability to sync featured versions from prod

### PR DESCRIPTION
**Pull request by peterbe**
@AdrianGaudebert r?

So this is the least hacky thing I can image we start with. I first tried to write it as a script that anybody could own and run with an API token. But then I needed a new permission (and possibly tie that to a group) and our FeaturedReleases model couldn't accept anything other than multipart form.

I say we make this available and it only shows on stage or localhost and it'll make stage look more sane and should solve the problem for QA.

### CODE REVIEW

File: webapp-django/crashstats/manage/static/manage/js/featured-versions.js [hunk 1]
```diff
@@ -19,8 +19,40 @@
...
+                $('.featured-on-prod button').on('click', function() {
+                    $('input[type="checkbox"]:checked').attr('checked', false);
+                    $('tbody tr').each(function() {
+                        var product = $('td', this).eq(0).text();
+                        var version = $('td', this).eq(1).text();
+                        if (featured[product].indexOf(version) > -1) {
+                            $('input[type="checkbox"]', this).attr('checked', true);
+                        }
+                    });
+                    $('.featured-on-prod p:hidden').show(600);
```
Comment (AdrianGaudebert) [inline webapp-django/crashstats/manage/static/manage/js/featured-versions.js]: This doesn't work, it's not ``hidden`` as far as I can tell.

**Comment by AdrianGaudebert** [comment]
It looks good to me. The filtering on the domain name is kinda weird, but I agree it looks like a very simple solution. What worries me is that it still requires a manual step, and I'm afraid we might forget to do it. Or QA will see a failing test and have to go on stage to fix the featured version before re-running the test. It's still a better solution, but clearly not perfect yet.

File: webapp-django/crashstats/manage/static/manage/js/featured-versions.js (hunk 1 above)
Comment (peterbe) [inline webapp-django/crashstats/manage/static/manage/js/featured-versions.js]: Will fix

**Comment by AdrianGaudebert** [comment]
r+

File: webapp-django/crashstats/manage/static/manage/js/featured-versions.js (hunk 1 above)
Comment (peterbe) [inline webapp-django/crashstats/manage/static/manage/js/featured-versions.js]: Actually, the `p` tag looked like this:
```
<p style="display:none">
    Checkboxes updated to match <code>crash-stats.mozilla.com</code>.<br>
    Verify and press <b>Update Featured Versions</b> below.
</p>
```

**Comment by peterbe** [comment]
nits addressed. Going to merge.

**Pull request merged**